USSF-ORBIT/ussf-portal-prototype
# Set up prettier & eslint

## Description 

This PR adds & configures both [prettier](https://prettier.io/) and [eslint](https://eslint.org/) on this project, to play well with our existing setup (TypeScript, NextJS).

* Add sane defaults based on other Truss projects (imports, a11y, security)
* Update ignore files
* Update packages
* Commit changes to existing code based on prettier config

Fixes #7  

## Review Notes

To test, you can check this branch and run:

* `yarn install` to install new packages
* `yarn format` will run Prettier and _write changes to files_ (there should be none)
* `yarn lint` will run TypeScript compile to check for compilation errors, and `ESLint` and output warnings/errors without fixing (there will be some)
